### DISCUSSION TITLE
What is the bridge flow ratio related to ?

### DISCUSSION THREAD

**Discussion by DzzD** [supermerill/SuperSlicer#2094] [Q&A] [answered]
Helllo,

I have trouble to understand what is the real bridge flow ratio when it is set to 100% under "With & Flow>Flow ratio>Bridge" because looking at the gcode preview with mode  "volumetric flow ratio" the bridge volumetric flow is a lot higer than other part that are printed at the same speed width the same 100% flow, So I assume that 100% for bridge flow ratio represent more than 100% for other part ?

So my question is what is the bridge flow ratio % related to ?

**Comment by neophyl** [comment, marked as answer]
At the moment bridges are assumed to be printed at a layer height = nozzle width, not your actual specified layer height.  This is because the extrusion has nothing/insufficient below it to get any squish.  As such if you keep the speed the same then the volumetric has to be increased.
Its why bridge layers always look 'fat' on the previews too as they are for example 0.4mm high unlike the rest of the layer.  Change the preview to height and you will see what I mean.

This logic for bridging extrusions has been a part of Slic3r since the very early days and is also part of Prusa Slicer up to the 2.3.x versions.  The new PS2.4 versions have an optional tickbox which basically turns off that logic.  Mainly due to user demand.

**Reply by supermerill** [reply]
the 100% flow is `(nozzle diameter)² * PI/4 mm³/mm`

**Reply by DzzD** [reply]
thanks, the answer is very clear, (It may be interresting to add the formula to the tooltip help one day)
